src/App.css

```diff
@@ -7,32 +7,30 @@
   pointer-events: none;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
+.images {
+  display: block;
+  max-width:230px;
+  max-height:95px;
+  width: auto;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
-}
+.reviewOnHomepage{
+  background-color: rgb(252, 242, 203);
+  border-style: solid;
+  border-color: rgb(211, 193, 37);
+  text-align: center;
+  
+
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
 }
+
+.flex {
```

Comment:
Consider an alternate class name for this - it should generally represent/ identify the elements that it applies to eg. review-list or something like that
You may also consider centering the cards that appear in this flex box - on a slimmer mobile view they all appear on the left hand side